options.go

```diff
@@ -309,6 +312,39 @@ func LabelLimitValueSize(size int) Option {
 	}
 }
 
+// SelfEvictDisablePing configures if ringpop should disable actively pinging
+// other members when it evicts itself from the network. It is not advised to
+// disable the ping as it might cause the suspicion time to kick in as chances
+// are high that the node will be shutdown before its self-eviction is gossiped.
+// Only use this setting if you explicitly require no extra network traffic
+// during shutdown.
+func SelfEvictDisablePing(disabled bool) Option {
```

Comment:
Not really except for exposing it to users in case there is a need to turn it off in the future. It is more of a failsafe to have it than a functional piece at the moment.